tox.ini

```diff
@@ -17,22 +18,38 @@ commands =
     pytest -vv
 
 [testenv:flake8]
+skip_install = True
 deps =
     flake8
 commands = flake8 {posargs}
 
-[testenv:docs]
+[testenv:check-manifest]
+skip_install = True
 deps =
-    -r{toxinidir}/docs/requirements_docs.txt
+   check_manifest
 commands =
-    python setup.py build_sphinx -W
+   check-manifest -v
```

Comment:
I wonder if it would make sense to include this in `package`, because checking the MANIFEST.in is related to packaging. Or do we want to keep each tool in a separate tox environment, regardless of what "topic" they are related to?

Reply:
btw, there seems to be a space missing in the indentation here (it should be 4 spaces but is only 3 spaces).

Reply:
Including in package is a reasonable suggestion. I can't think of an obvious benefit to keeping the two separate, and there will be a speed benefit to merging them (one less VM to spin up), so I'll move it over.